### PR TITLE
refactor: (GenAI) Reorganized Extensions Samples (Group A)

### PR DESCRIPTION
**Pull request by Thoughtseize1**
#### Path for all new files is `/generative_ai/extensions/`
|  Region Tag	| Old file | New File  	| 
|---					|---			|---				|
|		generativeaionvertexai_delete_extension		|	gemini_extensions.py	|	delete_example.py	|
|		generativeaionvertexai_create_extension		|	gemini_extensions.py	|	create_example.py	|
|		generativeaionvertexai_execute_extension		|	gemini_extensions.py	|	execute_example.py	|
|		generativeaionvertexai_get_extension		|	gemini_extensions.py	|	get_example.py	|
|		generativeaionvertexai_list_extensions		|	gemini_extensions.py	|	list_example.py	|

❗**Files need to be deleted after approval:**
`generative_ai/gemini_extensions.py`
`generative_ai/gemini_extensions_test.py`

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/extensions/create_example.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                "open_api_gcs_uri": "gs://vertex-extension-public/code_interpreter.yaml"
+            },
+            "auth_config": {
+                "google_service_account_config": {},
+                "auth_type": "GOOGLE_SERVICE_ACCOUNT_AUTH",
+            },
+        },
+    )
+    print(extension.resource_name)
+    # Example output:
```
Comment (msampathkumar) [inline generative_ai/extensions/create_example.py:46]: Minor Nit:  can you use & updates other too please?
```suggestion
    # Example response:
```

In the future, when we conduct a `grep` search for "`Example response`", we can promptly locate the samples that lack the same response.

File: generative_ai/extensions/execute_example.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    extension = extensions.Extension(extension_id)
+
+    response = extension.execute(
+        operation_id="generate_and_execute",
+        operation_params={"query": "find the max value in the list: [1,2,3,4,-5]"},
+    )
+    print(response)
+    # Example output:
```
Comment (msampathkumar) [inline generative_ai/extensions/execute_example.py:36]: Minor Nit:  can you use & updates other too please?
```suggestion
    # Example response:
```

In the future, when we conduct a `grep` search for "`Example response`", we can promptly locate the samples that lack the same response.

Comment (Thoughtseize1) [reply generative_ai/extensions/execute_example.py]: I'll do it! Thank you for feedback👍

File: generative_ai/extensions/create_example.py (hunk 1 above)
Comment (Thoughtseize1) [reply generative_ai/extensions/create_example.py]: Agree with you! Fixed it👍

**Review by msampathkumar — APPROVED** [review]

**Pull request merged**